setup.py

```diff
@@ -35,31 +34,34 @@
 
 PY3 = sys.version_info[0] == 3
 
-with io.open('README.md', encoding='utf-8') as f:
-    long_description = f.read()
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
 
+try:
+    with io.open('README.md', encoding='utf-8') as f:
+        long_description = f.read()
+except FileNotFoundError:
+    long_description = ''
 
-class Tox(TestCommand):
-    user_options = [('tox-args=', None, "Arguments to pass to tox")]
+
+class CleanCommand(Command):
+    """Custom clean command to tidy up the project root."""
+    user_options = []
 
     def initialize_options(self):
-        TestCommand.initialize_options(self)
-        self.tox_args = ''
+        pass
```

Comment:
Do we still need these functions if they do nothing?

Reply:
For me it's still TODO. I have not looked at some setup.py functions (tests for one) too closely. I will take a look on cleaning it up after i got all the tests green .